metricbeat/module/apache/apache.asciidoc

```diff
@@ -15,5 +15,21 @@ The Apache Module has currently one MetricSet which is called status. It fetches
 === Apache Status Metricset
 
 
-TODO
-* List available selectors
+The apache status metricset offers the following selectors:
+
+* uptime
+* cpu
+* connections
+* load
+* scoreboard
+
+In addition the following top fields can be used as selector:
```

Comment:
`s/selector/selectors/`, but I would just remove this sentence and create a single list.